chapters/errors.Rmd

```diff
@@ -11,43 +11,37 @@ those programs might still contain bugs.
 We should therefore plan from the start to detect and handle errors.
 
 Something that goes wrong while a program is running
-is sometimes referred to as an \gref{exception}{exception} from normal behavior.\index{exception}
+is sometimes referred to as an exception from normal behavior.
```

Comment:
I think this is the first use of 'exception', so index it even if we don't glossary-reference it?

Reply:
I wasn't sure whether to glossary reference and index here in the introduction or in the following section which is titled "exceptions" (or both)?